PolarisO365.ps1

```diff
@@ -767,6 +767,229 @@ function Get-PolarisO365OneDrive() {
     return $user_details
 }
 
+function Get-PolarisO365SharePoint() {
+    <#
+    .SYNOPSIS
+    Returns a filtered list of O365 SharePoint sites and/or document libraries for a given subscription in a given Polaris instance.
+    .DESCRIPTION
+    Returns a filtered list of Office 365 SharePoint sites and/or document libraries from a given subscription and Polaris instance, taking
+    an API token, Polaris URL, subscription ID, and search string.
+    .PARAMETER Token
+    Polaris API Token.
+    .PARAMETER PolarisURL
+    The URL for the Polaris instance in the form 'https://myurl'
+    .PARAMETER SubscriptionID
+    The Polaris subscription ID for a given O365 subscription. Can be obtained with the
+    'Get-PolarisO365Subscriptions' command.
+    .PARAMETER SearchString
+    Search string, used to filter site or drive name.
+    .PARAMETER Includes
+    It indidates if the returned object includes only SharePoint sites, drives or both. The value can only be 'SitesOnly', 'DocumentLibrariesOnly', 'Both'.
+    .INPUTS
+    None. You cannot pipe objects to Get-PolarisO365SharePoint.
+    .OUTPUTS
+    System.Object. Get-PolarisO365SharePoint returns an array containing the ID, Name,
+    email address, and SLA details for the returned O365 OneDrive users.

```

Comment:
Please update the comment. This should be SharePoint objects, not OneDrive users.